src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -445,7 +446,7 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
     priorityContext = TaskPriority;
 
     let firstEffect;
-    if (finishedWork.effectTag !== NoEffect) {
+    if (finishedWork.effectTag > Bailout) {
```

Comment:
This was not necessary (it's a root) but I just did this for consistency so I can remove `NoEffect` import here. I can put this back.